bundle/manifests/search-v2-operator.clusterserviceversion.yaml

```diff
@@ -232,14 +236,6 @@ spec:
                 - --leader-elect
                 command:
                 - /manager
-                image: quay.io/stolostron/search-v2-operator:2.7.0-dc0fc8c59239b6a0a3f440b7752101523d87f597
```

Comment:
**Note to reviewers:** This section got moved below, contents are exactly the same.